docs/deploy/create_server.rst

```diff
@@ -190,11 +203,13 @@ Hostnames follow the format ``ocp##.open-contracting.org`` (ocp01, ocp02, etc.).
 
       If the DNS records have not yet propagated, you can temporarily use the server's IP address instead of its hostname in the roster.
 
-#. Run the `onboarding <https://github.com/open-contracting/deploy/blob/main/salt/onboarding.sls>`__ and core state files, which upgrade all packages, configure the hostname, and apply the base configuration. Replace ``TARGET`` and ``ocpXX``:
+#. :doc:`../develop/update/network`.
+
+#. Run the `onboarding <https://github.com/open-contracting/deploy/blob/main/salt/onboarding.sls>`__ and core state files, which upgrade all packages, configure the hostname and apply the base configuration.
 
    .. code-block:: bash
 
-      salt-ssh --log-level=trace TARGET state.apply 'onboarding,core*' pillar='{"host_id": "ocpXX"}'
+      salt-ssh --log-level=trace TARGET state.apply 'onboarding,core*'
```

Comment:
Should the create_server process also link to https://ocdsdeploy.readthedocs.io/en/latest/develop/update/firewall.html?

Reply:
Yes, I've just added a "Firewall" step linking to these docs. 
It should only be needed for docker deployments.